File: app/src/main/java/ch/zuehlke/sbb/reddit/data/source/remote/model/RedditNewsAPIResponse.java
package ch.zuehlke.sbb.reddit.data.source.remote.model;

import com.google.gson.annotations.Expose;

/**
 * Created by chsc on 12.11.17.
 */

public class RedditNewsAPIResponse {

    @Expose
    private RedditNewsAPIResponseData data;
    @Expose
    private String kind;


    public RedditNewsAPIResponseData getData() {
        return data;
    }

    public void setData(RedditNewsAPIResponseData data) {
        this.data = data;
    }

    public String getKind() {
        return kind;
    }

    public void setKind(String kind) {
        this.kind = kind;
    }
}

File: app/src/main/java/ch/zuehlke/sbb/reddit/data/source/remote/model/news/RedditNewsAPIResponse.kt
package ch.zuehlke.sbb.reddit.data.source.remote.model.news

import com.google.gson.annotations.Expose

/**
 * Created by chsc on 12.11.17.
 */

class RedditNewsAPIResponse {

    @Expose
    var data: RedditNewsAPIResponseData? = null
    @Expose
    var kind: String? = null
}
